tests/unit/package_managers/bundler/test_main.py

```diff
@@ -119,7 +119,7 @@ def test_get_main_package_name_and_version_from_repo_without_origin(
     assert "Failed to extract package name from origin remote" in exc_info.value.friendly_msg()
 
 
-def test__prepare_for_hermetic_build_injects_necessary_variable_into_empty_config(
```

Comment:
I know these changes correspond to your guideline change, but I think the easiest is to just use the same scheme everywhere, because after this patch there are still the following violators (yes, mine, from a recent golang changes):
```
tests/unit/package_managers/test_gomod.py

test__init__
test__init__fail
test__bool__
```

Reply:
Yes, I have left those intact intentionally, because a function name starts with two underscores (in my naming scheme a hypothetical test for any these three would have started with triple underscore).

Reply:
I understand, but if this patch is to have any tangible meaning then we should apply it globally, no exceptions, pure consistency and be done with it :) .

Reply:
Fair enough.